config/motorola-nora.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+# NOTE: we have the DTB for nora, but in downstream DTS
+# we see that these come from msm8917-jeter-common.dtsi
+OPTIONS=()
+PANELS=(
+	[tianma_570_hd_video_v0]="motorola,jeter-570-tianma"
+	[tianma_571_hd_video_v0]="motorola,jeter-ili9881h-tianma"
```

Comment:
```suggestion
	[tianma_571_hd_video_v0]="motorola,jeter-571-tianma"
```
I would prefer controler ic name, but in your case we only know this panel. Maybe better to keep these consitent.

Reply:
honestly same but i'll have to change this in lk2nd as well ><